mining.go

```diff
@@ -608,7 +604,7 @@ func minimumMedianTime(best *blockchain.BestState) time.Time {
 // medianAdjustedTime returns the current time adjusted to ensure it is at least
 // one second after the median timestamp of the last several blocks per the
 // chain consensus rules.
-func medianAdjustedTime(best *blockchain.BestState, timeSource blockchain.MedianTimeSource) time.Time {
+func (g *BlkTmplGenerator) medianAdjustedTime(best *blockchain.BestState, timeSource blockchain.MedianTimeSource) time.Time {
```

Comment:
I think this could be simplified further, `best` can be fetched from `g.chain.BestSnapshot()` as well as the time source. The function signature would be simplified to `func (g *BlkTmplGenerator) medianAdjustedTime() time.Time` with these modifications.

Reply:
Good catch, thanks.  Fixed